### PR TITLE
Kill switch for service worker

### PR DESCRIPTION
**Pull request by bogas04**
* [ ]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [ ]  Ran `npm test` & fixed newly introduced lint errors.
* [ ]  Checked console for errors.

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: server/template.js [hunk 1]
```diff
@@ -1,37 +1,84 @@
 import marinate from 'marinate';
 import { ONLINE_COLOR } from '../common/constants';
 
+const preloadImages = [
```
Comment (bogas04) [inline server/template.js]: Right now these are hard coded but eventually we would be reading these from a manifest file generated by webpack.

Comment (plug-n-play) [reply server/template.js]: Do we already have a issue created for the follow-up item we need to do?

File: server/template.js [hunk 2]
```diff
@@ -1,37 +1,84 @@
...
+const preloadImagesHTML = preloadImages
+  .map(i => `<link async rel="preload" href="${i}" as="image">`)
+  .join('\n');
+
+const preloadFonts = [
+  '/assets/fonts/AnmolLipiSG.ttf?v=1',
+  '/assets/fonts/GurbaniAkharHeavyTrue.ttf?v=1',
+];
+const preloadFontsHTML = preloadFonts
+  .map(f => `<link async rel="preload" href="${f}" as="font"></link>`)
```
Comment (navdeepsinghkhalsa) [inline server/template.js]: Is there any reason this isn't self-closing?

Comment (bogas04) [reply server/template.js]: Nice catch, thank you!

**Pull request merged**